webserver/include/webserver/pluginhelper.h

```diff
@@ -27,25 +28,27 @@ inline void registerPluginSimple(
   ServiceFactoryT serviceFactory_,
   const std::string& serviceName_)
 {
+  namespace fs = boost::filesystem;
   namespace po = boost::program_options;
 
-  const util::WorkspaceOptions workspaces
-    = util::parseConfigFile(vm_["workspaceCfgFile"].as<std::string>());
-
-  // Create a handler instance for all workspaces.
-  for (const auto& workspace : workspaces)
+  for (fs::directory_iterator it(vm_["workspace"].as<std::string>());
+    it != fs::directory_iterator();
+    ++it)
   {
-    const std::string wsId = workspace.first;
-    const util::WorkspaceOption& wsOpt = workspace.second;
+    std::string project = it->path().filename().native();
+
+    std::string connStr = util::updateConnectionString(
```

Comment:
What if the user already renamed the database or removed it. We will throw a `runtime error`. Is it good?
I think we should only skip the project and write a log message that database doesn't exist.